### DISCUSSION TITLE
Is rx-query still active?

### DISCUSSION THREAD

**Discussion by Deewens** [timdeschryver/rx-query#33] [Q&A] [answered]
Hi,

I am looking for a library like this one for a professional Angular application, so I would like to know if I can use this library in a production app' and if the library is still in active development?

Thanks!

**Comment by timdeschryver** [comment, marked as answer]
Hi @Deewens , I'm not really maintaining the library anymore (due to some factors).
I did however recently started a similar library, but better suited and aimed for Angular applications, called [ngx-query-state](https://github.com/timdeschryver/query-state).
ngx-query-stateis still fresh, so things can change, and I also can't promise how fast the development cycle will be. If you like the library, feel free to help.

**Reply by Deewens** [reply]
Okay, I see! Thanks for your answer! 
Would be glad to help if I get the time
